INSTALL.md

```diff
@@ -1,5 +1,5 @@
 ## Build Instructions
 
-1. Obtain a clean DOL of PBR PAL 1.0 and place it in the build folder and name it `baserom.dol`.
+1. Obtain a clean DOL of PBR PAL 1.0 and place it in the include/ folder and name it `baserom.dol`.
```

Comment:
baserom needs to be in the base folder (pokerevo), not build or include

Reply:
How's this?